### PR TITLE
Change language

### PR DESCRIPTION
**Pull request by LeonZhang7**
added correct routing for changelanguage
- create change language page
- added routing
adding change language
- adding google change language element
Remove the the extra language bar in /head
Update index.html
- changing the placement of the translate
Added footer
- Added footer for the app
Update Footer.js
- taking off + from return
taking off styling
- taking off styling for change language button
adding the script to the footer
- adding change language script to the footer
fixing language component bug
- reactivity is now fixed only minor styling needs to be added
adding final fixes to the language changes
- how the footer is reactive and  the change language button only appears once
deleted uneeded footer class
- adding correct implementation of footer and minor styling
- last styling still needs to be added
Added 100 vh to pages to make the footer be lower on the screen
Changed page sizes for multiple pages
- Changed page sizes for multiple pages to fit the translate element at the end of the page

### CODE REVIEW

**Review by fatema-a — APPROVED** [review]
adding style changes in the pages and change language everything looks good

File: frontend/src/components/Settings/ProfileInfoSetting.js [hunk 1]
```diff
@@ -43,75 +43,77 @@ function ProfileInfoSettings() {
...
+						<div className="containRequest">
+					<h5 className="requests">Name & location</h5>
+					<div className="arrow">
+					<Link to="/NameSetting">
+						<img src={arrow} alt="node" />
+					</Link>
+					</div>
+				</div>
+				<hr></hr>
+				<div className="containRequest">
```
Comment (Chovv) [inline frontend/src/components/Settings/ProfileInfoSetting.js:97]: I see you added a redirect to the language tab.

File: frontend/public/index.html [hunk 2]
```diff
@@ -51,5 +64,22 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+
+    
   </body>
+
+
+  <footer class="index-footer">
```
Comment (Chovv) [inline frontend/public/index.html:72]: I see you added the language change feature using google translate.

File: frontend/src/App.js [hunk 3]
```diff
@@ -209,6 +214,14 @@ function App() {
             </Protection>
           }
         />
+        <Route
```
Comment (Chovv) [inline frontend/src/App.js:217]: Added protection to the routing, I like it.

**Review by Chovv — COMMENTED** [review]
Overall solid implementation, with some responsiveness fixes here and there.

**Pull request merged**